src/data-display/board-item/type.ts

```diff
@@ -8,5 +8,6 @@ export type ButtonEventHandler = (...args: any[] | any) => Promise<void> | void;
 
 export interface IconSet {
     iconName: string;
+    tooltipText?: string;
```

Comment:
nitpick: I think it would be safer to specify it as a `TranslationResult` type.

Reply:
Thanks!!